toolchain/semantics/semantics_ir.cpp

```diff
@@ -47,10 +47,10 @@ auto SemanticsIR::MakeBuiltinIR() -> SemanticsIR {
 auto SemanticsIR::MakeFromParseTree(const SemanticsIR& builtin_ir,
                                     const TokenizedBuffer& tokens,
                                     const ParseTree& parse_tree,
-                                    llvm::raw_ostream* verbose_stream)
+                                    llvm::raw_ostream* vlog_stream_)
```

Comment:
Do you need to use the trailing `_` here and elsewhere in parameters? I thought the goal was to *only* support logging in class methods to start with?

Reply:
sed mistake; fixed